api/views/serializers.py

```diff
@@ -693,7 +693,8 @@ class Meta:
             "favorite_min_rating",
             "image_scale",
             "save_metadata_to_disk",
-            "datetime_rules"
+            "datetime_rules",
```

Comment:
I've added both datetime_rules and default_timezone here, but not sure if that is correct.